systemtest/src/test/java/io/strimzi/systemtest/AbstractClusterIT.java

```diff
@@ -168,4 +170,10 @@ public String consumeMessages(String clusterName, String topic, int groupID, int
                             "-kafka:9092 --topic " + topic + " --group-id " + groupID + " & sleep "
                             + timeout + "; kill %1").out();
     }
+
+    void checkErrorsInLogCC() {
+        //TODO add blacklist for unexpected errors
+        String searchPattern = "\'Exception\\|Error\\|Throwable\'";
```

Comment:
The `'` quotes belong in `searchInLog()` – I don't think callers of the method should care that you happen to be passing the regex to grap via `bash -c` and therefore need to be quoted for the shell: It's an implementation detail of `searchInLog()`.  I suppose the same applies to the need to `\`-quote the `|`, but TBH it's not clear to me whether it's bash or grep which is handling that quoting – do you know?

Reply:
@tombentley TBH I tried to use many patterns (without `'` quotes, without `\|`) but in practice it turns out that if we want to use many keywords for executing `grep` through `Exec` we need to use all added symbols and it was a reason of using search pattern for it. But in another hand (as you said) the callers don't care about `'` quotes etc, so looks like the better way will be move regex logic to method `searchInLog()` if I understood you right